bootstrap.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+AUTOCONF_FILES="Makefile.in aclocal.m4 ar-lib autom4te.cache compile \
+        config.guess config.h.in config.sub configure depcomp install-sh \
+        ltmain.sh missing *libtool test-driver"
```

Comment:
This is the list of files created by autotools. Can you expand on what fails if these files are already present?

Reply:
I think it nothing will fail if it is not cleaned up before running the script again. This is just for cleanup after if we want to.